### DISCUSSION TITLE
Why retargeting only to .net framework

### DISCUSSION THREAD

**Discussion by ringwaldK** [oleg-shilo/wixsharp#1335] [Q&A]
Hello everyone,

question why is the version `2.0.1` retargeting .net framework? Please can you explain the reason. 

Thanks 👍🏼

**Comment by oleg-shilo** [comment]
It is because WiX4 has retargeted all managed assemblies that are involved in WiX build process (e.g. custom BA) to the later .NET framework. It is still a very conservative (non-latest) version: 4.7.1.
